### PR TITLE
[EDX-102] correct spacing on code text element

### PR DESCRIPTION
**Pull request by MymmiJ**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

A PR description indicating the purpose of the PR.
* [Jira ticket](https://ably.atlassian.net/browse/EDX-102).

## Review

There's quite a lot going on here for the small size of change, here's a summary:
* The user agent stylesheet was overriding some CSS because we were in quirks mode, this is because various <table> elements were trying to render text especially '/n/t'. This has been fixed: (table, tbody, tr).
* There's some reset styles in Ably UI that do quite a lot of base styling that we have to ignore in `global.css`
* The base font-family and font-weight have been wrong, that had to be corrected in the global CSS reset/normalize styles to pull through the changes we need to Code (Museo Sans/Menlo => NEXT Book/Arial)
* This messed with a few subsequent styles, mostly font-weight being weird/wrong (dd, td, dt, li) and Sidebar menus not appearing correctly.
* Once this is all fixed, the line-height can be successfully set and display as expected in the SyntaxHighlighter styles for 'hljs' classes & the code text element now displays correctly.

### CODE REVIEW

File: src/components/Sidebar/RightSidebar/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const mapMenuItemToSidebarItem = ({ name, id, level }) => ({
   level,
 });
 
-const RightSidebar = ({ menuData, className }) => {
+const RightSidebar = ({ menuData, languages, className }) => {
```
Comment (DuncanEdwards) [inline src/components/Sidebar/RightSidebar/index.js:11]: Convert to TS (if you get stuck I can help!)

File: src/components/Sidebar/index.js [hunk 2]
```diff
@@ -8,12 +8,13 @@ import { EXPAND_MENU } from './consts';
 const Sidebar = ({
```
Comment (DuncanEdwards) [inline src/components/Sidebar/index.js:8]: Convert to TS (if you get stuck I can help!)

**Review by DuncanEdwards — CHANGES_REQUESTED** [review]
I thought I'd stop at making you move two to TS! We should do stuff like the table and list components in bulk at some later point

File: src/components/Sidebar/index.js (hunk 2 above)
Comment (MymmiJ) [reply src/components/Sidebar/index.js]: Have had a go at TS definitions for this and RightSidebar now!

File: src/components/Sidebar/sidebar-props.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { EXPAND_MENU } from './expand-menu-enum';
```
Comment (DuncanEdwards) [inline src/components/Sidebar/sidebar-props.ts:1]: As it isn't used by more than one component, this doesn't need to be in its own file, can just be placed inline in Sidebar.

File: src/components/Sidebar/RightSidebar/right-sidebar-props.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import { MenuData } from './menu-data';
```
Comment (DuncanEdwards) [inline src/components/Sidebar/RightSidebar/right-sidebar-props.ts:1]: Only used by RightSidebar, so can just be inline in the `RightSidebar` component

**Review by DuncanEdwards — CHANGES_REQUESTED** [review]

File: src/components/Sidebar/sidebar-props.ts (hunk 3 above)
Comment (MymmiJ) [reply src/components/Sidebar/sidebar-props.ts:1]: This is also used by SidebarLinkItem, SidebarLinkMenu and left-sidebar.js as an enum now, replacing the frozen object 'enum' used before, should I still move it?

Comment (MymmiJ) [reply src/components/Sidebar/sidebar-props.ts:1]: Ah I understand you mean the whole file now after reading the other comment, got it!

**Review by DuncanEdwards — APPROVED** [review]

**Pull request merged**